### PR TITLE
Modify cluster verification to bypass proxy and directly talk to cluster apiserver

### PR DESCRIPTION
**Pull request by vinaykul**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
/kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**: Nginx proxy is flaky. Cluster verification script that talks to the apiserver via proxy intermittently fails and this can cause the cluster bringup script to take a long time to start or intermittently fail. Verifying directly against apiserver is a reliable way to ensure actual cluster is fine. The test will still use proxy as per architecture.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: cluster/gce/util.sh [hunk 1]
```diff
@@ -2362,7 +2362,8 @@ function kube-up() {
     if [[ "${KUBERNETES_SCALEOUT_PROXY:-false}" == "true" ]]; then
       echo "Scaleout proxy public IP: ${PROXY_RESERVED_IP}:8888"
       echo "Scaleout proxy internal IP: ${PROXY_RESERVED_INTERNAL_IP}:8888"
-      sed -i "s/server: https:\/\/.*/server: http:\/\/${PROXY_RESERVED_IP}:8888/" ${KUBECONFIG}
+      cp -f ${KUBECONFIG} ${LOCAL_KUBECONFIG}
+      sed -i "s/server: https:\/\/.*/server: http:\/\/${PROXY_RESERVED_IP}:8888/" ${LOCAL_KUBECONFIG}
```
Comment (yb01) [inline cluster/gce/util.sh:2366]: the local_kubeconfig was updated every time the create_kubemark-Master is called. we probably leave it untouched and if any logic is needed in the upper level function, i.e. in the start Kubemark.sh, they can update as needed.

Comment (vinaykul) [reply cluster/gce/util.sh:2366]: It is only updated when scaleout_proxy is true (first time kube-up is called in workflow that deploys proxy and RP)
start-kubemark does not have access to the variables in invoked bash functions. There is no way to export it back and handle this in caller.

**Review by yb01 — APPROVED** [review]
/lgtm

**Pull request merged**